src/responder/common/cache_req/cache_req.c

```diff
@@ -1275,6 +1275,15 @@ static void cache_req_input_parsed(struct tevent_req *subreq)
     state = tevent_req_data(req, struct cache_req_state);
 
     ret = sss_parse_inp_recv(subreq, state, &name, &domain);
+
+    if (state->domain_name && domain && strcmp(state->domain_name, domain)){
```

Comment:
```c
if (state->domain_name != NULL && domain != NULL && strcmp(state->domain_name, domain) != 0) {
```

Ack otherwise.

Reply:
Thank you for noticing, it's hard to follow the coding style when it differs in the projects that I'm working on.

Reply:
Fixed in the newly pushed revision.